src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/scatterPlot.js

```diff
@@ -173,16 +170,17 @@ class ScatterPlot extends Visualization {
      * Adds panning and zooming functionality to the visualization.
      */
     addPanAndZoom(box_width, box_height, svg, margin, scaleAndAxis, scatter, points) {
-        let zoomClass = 'zoom'
-        let minScale = 0.5
-        let maxScale = 20
+        const minScale = 0.5
+        const maxScale = 20
+        const right_button = 2
+        const mid_button = 1
+        const scroll_wheel = 0
```

Comment:
Probably my fault, but the variable name convention differs between names. Should probably all be snakeCase. 
Maybe those constants should also be at the top of the file in the constants section.